Makefile.am

```diff
@@ -958,6 +958,7 @@ EXTRA_DIST += $(motr_cron_SCRIPTS)
 motr_confdir = $(motrdir)/conf
 motr_conf_DATA = scripts/install$(motr_confdir)/motr.conf \
                  scripts/install$(motr_confdir)/setup.yaml \
+                 scripts/install$(motr_confdir)/support.yaml \
```

Comment:
I am not knowing much of MOTR code & its install process.
@mehjoshi please check if this line will bundle & install support.yaml

Reply:
@sachinpunadikar : This command will include the support.yaml file in motr rpms.